plookup/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "plookup"
+version = "0.1.0"
+authors = ["wborgeaud <williamborgeaud@gmail.com>"]
+edition = "2018"
+
+[dependencies]
+plonky = {path="../../plonky"}
```

Comment:
Can this be `path = ".."`? I sometimes use copies with other names, like `plonky-master` for a copy that I keep on master.

Reply:
Yes that works too. Made the change.